src/Gui/QSint/actionpanel/actiongroup.cpp

```diff
@@ -243,21 +211,19 @@ QString ActionGroup::headerText() const
     return myHeader->myTitle->text();
 }
 
-void ActionGroup::setHeaderText(const QString & headerText)
+void ActionGroup::setHeaderText(const QString &headerText)
 {
     myHeader->myTitle->setText(headerText);
 }
 
-void ActionGroup::setHeaderIcon(const QPixmap& icon)
```

Comment:
Let's avoid changing `&` and `*` forward on - else we'll move it back and forth each each time a new dev takes a look, causnig git blame cancer